app/Http/Middleware/CheckIfTeacher.php

```diff
@@ -4,10 +4,11 @@
 
 use Closure;
 use Illuminate\Http\Request;
-use App\Models\Teacher;
+use App\Models\Employee;
+use App\Models\Role;
 use Illuminate\Support\Facades\Auth;
 
-class ChekcIfTeacher
+class CheckIfTeacher
```

Comment:
Het bestand heet ook nog `ChekcIfTeacher.php` in plaats van `CheckIfTeacher.php`

Reply:
Hij moet ook checkifteacher heten. Had m een keer verwijderd en toegevoegd